### PR TITLE
Adding 2018.4 version of testing project.

### PR DESCRIPTION
**Pull request by vincent-unity**
Adding 2018.4 version of testing project. as the package setup is quite different to 2019.3.
For example, all the ide packages like `rider`, `vs code`, `visual studio` are not available on 2018.4, also the package manager UI itself is a package on 2018.4 as well...

### CODE REVIEW

File: TestProjects/NotificationSamples_2018/Assets/Scripts/AndroidManualTests.cs [hunk 1]
```diff
@@ -0,0 +1,328 @@
+#if PLATFORM_ANDROID
```
Comment (todi1856) [inline TestProjects/NotificationSamples_2018/Assets/Scripts/AndroidManualTests.cs]: Isn't this a duplicate of TestProjects/NotificationSamples/Assets/Scripts/AndroidManualTests.cs

If so, why not put it in com.unity.mobile.notifications/com.unity.mobile.notifications/Tests/ folder?

Or use some other trick, like create a local package for file sharing

File: TestProjects/NotificationSamples_2018/Assets/Scripts/RemoteNotificationsManualTests.cs [hunk 2]
```diff
@@ -0,0 +1,130 @@
+#if UNITY_IOS
+using System;
```
Comment (todi1856) [inline TestProjects/NotificationSamples_2018/Assets/Scripts/RemoteNotificationsManualTests.cs]: duplicating file, not good

File: TestProjects/NotificationSamples_2018/Assets/Scripts/AndroidManualTests.cs (hunk 1 above)
Comment (vincent-unity) [reply TestProjects/NotificationSamples_2018/Assets/Scripts/AndroidManualTests.cs]: yeah... I agreed. Will find a better way to do this.

File: TestProjects/NotificationSamples_2018/Assets/Scripts/RemoteNotificationsManualTests.cs (hunk 2 above)
Comment (vincent-unity) [reply TestProjects/NotificationSamples_2018/Assets/Scripts/RemoteNotificationsManualTests.cs]: Agreed.

**Comment by vincent-unity** [comment]
Close it as @renaldasrep is working on a new sample.

**Pull request closed without merging**